webapp/src/main/resources/db/migration/V71__Add_branch_source.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE tm_text_unit_to_branch(
+   id bigint AUTO_INCREMENT PRIMARY KEY,
+   tm_text_unit_id bigint NOT NULL,
+   branch_id bigint NOT NULL
+);
+
+ALTER TABLE tm_text_unit_to_branch
+ADD CONSTRAINT FK__TM_TEXT_UNIT_TO_BRANCH__TM_TEXT_UNIT_ID FOREIGN KEY (tm_text_unit_id) REFERENCES tm_text_unit(id);
+
+ALTER TABLE tm_text_unit_to_branch
+ADD CONSTRAINT FK__TM_TEXT_UNIT_TO_BRANCH__BRANCH_ID FOREIGN KEY (branch_id) REFERENCES branch(id);
+
+create index I__TEXT_UNIT_TO_BRANCH__TEXT_UNIT_ID on tm_text_unit_to_branch (tm_text_unit_id);
+
+CREATE TABLE branch_source(
+   id bigint AUTO_INCREMENT PRIMARY KEY,
+   branch_id bigint NOT NULL,
+   url VARCHAR(300) NOT NULL
```

Comment:
I'm curious. How did you decide on this number?

Reply:
That was the initial number I picked for testing, I changed the number to 255 to match other VARCHAR values used in the db. @byronantak